alphadia/workflow/peptidecentric.py

```diff
@@ -260,6 +260,30 @@ def norm_to_rt(
         else:
             raise ValueError(f"Unknown norm_rt_mode {mode}")
 
+    @property
```

Comment:
are these properties also to be accessed from outside the class?

Reply:
No, most likely not.

Reply:
then I would suggest settings these values as instance variables or simple methods